submissions/kuslosolodka/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,80 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+  species: "dog",
+  name: "Brovko",
+  gender: "male",
+  legs: 4,
+  hands: undefined,
+  saying: "woof-woof!",
+};
+const cat = {
+  species: "cat",
+  name: "Murka",
+  gender: "female",
+  legs: 4,
+  hands: undefined,
+  saying: "meow!",
+};
+const woman = {
+  species: "human",
+  name: "Stefania",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  saying: "I am Stefania!",
+};
+const man = {
+  species: "human",
+  name: "Marko",
+  gender: "male",
+  legs: 2,
+  hands: 2,
+  saying: "I am Marko!",
+};
+const cat_woman = {
+  species: "supernatural",
+  name: "Batgirl",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  saying: cat["saying"],
+};
+dog.friend = `${woman.name},${man.name},${cat.name}`;
```

Comment:
`friend` should be plural, I guess. And it should be an array since here is where you keep your "raw" data, a model if you will. Then some other code should pick this data and format it to display.